lib/acts_as_taggable_on_mongoid/models/concerns/tag_methods.rb

```diff
@@ -34,8 +38,9 @@ def find_or_create_all_with_like_by_name(tag_definition, *list)
             end
           end
 
-          def create_tag(tag_definition, name)
+          def create_tag(tag_definition, owner, name)
```

Comment:
Doesn't depend on instance state (maybe move it to another class?) (UtilityFunction)